instrumentation/runtime/runtime.go

```diff
@@ -108,6 +108,7 @@ func Start(opts ...Option) error {
 
 func (r *runtime) register() error {
 	startTime := time.Now()
+	// DEPRECATED: This metric is deprecated in favor of processUptime
```

Comment:
I don't think this is enough. AFAIK, it won't show in godoc, as `// DEPRECATED` is for methods and structs.
I'm not finding any precedent where a metric was deprecated.
But how about specifying it in the metric description?

Reply:
Just edited the metric description to indicate the metric is deprecated. Any other corrections @dmathieu